packages/backend/src/models/ProjectModel/ProjectModel.test.ts

```diff
@@ -77,4 +79,62 @@ describe('ProjectModel', () => {
 
         expect(tracker.history.update).toHaveLength(1);
     });
+
+    describe('should update outdated metric filters in explores', () => {
+        const fieldRefsToTest: {
+            fieldRef: string;
+            tableName: string;
+            isOutdated: boolean;
+        }[] = [];
+        const convertedExplore =
+            ProjectModel.convertMetricFiltersFieldIdsToFieldRef(
+                mockExploresFromCache[0],
```

Comment:
can remove the other explores and just keep the first one in `ProjectModel.mock`. What do we think?